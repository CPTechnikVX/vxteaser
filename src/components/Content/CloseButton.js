import React     from 'react';
import PropTypes from 'prop-types';

function CloseButton({onCloseFn}) {
	return <div className={'vxteaser-control--close'} onClick={onCloseFn} />;
}

CloseButton.propTypes = {
	onCloseFn: PropTypes.func,
};

export default CloseButton;
