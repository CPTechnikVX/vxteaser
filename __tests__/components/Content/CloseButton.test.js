import React       from 'react';
import {render}    from '@testing-library/react';
import CloseButton from '../../../src/components/Content/CloseButton';

describe('render button', () => {
    test('empty', () => {
        const {container} = render(
                <CloseButton />
        );
        expect(container.firstChild).toMatchSnapshot();
    });
});
