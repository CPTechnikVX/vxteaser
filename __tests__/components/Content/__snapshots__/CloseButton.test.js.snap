// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`render button empty 1`] = `
<div
  class="vxteaser-control--close"
/>
`;
